### PR TITLE
Autofix RoundaboutMissingTag

### PR DESCRIPTION
**Pull request by vladlemberg**
### Description:

Convert RoundaboutMissingTag Check to AutoFix https://github.com/osmlab/atlas-checks/issues/416

### Potential Impact:

This Check is no longer available for regular editing after conversion to AutoFix

### Unit Test Approach:

Verify that flag include the AutoFix suggestion

### Test Results:

Passed

### CODE REVIEW

File: src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/RoundaboutMissingTagCheck.java [hunk 1]
```diff
@@ -110,9 +113,15 @@ public boolean validCheckForObject(final AtlasObject object)
 
         if (maxOffendingAngles.isEmpty() && minOffendingAngles.isEmpty())
         {
-            this.markAsFlagged(object.getOsmIdentifier());
```
Comment (seancoulter) [inline src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/RoundaboutMissingTagCheck.java:113]: Should we keep this line?

**Review by seancoulter — COMMENTED** [review]
Looks great, just one question

File: src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/RoundaboutMissingTagCheck.java (hunk 1 above)
Comment (sayas01) [reply src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/RoundaboutMissingTagCheck.java:113]: Good call. I think we should.

Comment (vladlemberg) [reply src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/RoundaboutMissingTagCheck.java:113]: fixed

**Review by sayas01 — APPROVED** [review]
👍

**Review by seancoulter — APPROVED** [review]

**Review by sayas01 — APPROVED** [review]

**Pull request merged**